Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Buffer/Buffer.h

```diff
@@ -66,7 +65,7 @@ namespace AZ
             //! This function is only used for buffer created in host such as dynamic buffer which content is rewritten every frame
             bool OrphanAndUpdateData(const void* sourceData, uint64_t sourceDataSizeInBytes);
 
-            void* Map(size_t byteCount, uint64_t byteOffset);
+            AZStd::vector<void*> Map(size_t byteCount, uint64_t byteOffset);
```

Comment:
This needs a comment about the method itself explaining why this is a vector for addresses